docs/markdown/Helm/helm-overview.md

```diff
@@ -63,6 +63,8 @@ Created src/helm/example/BUILD:
   - Add helm_chart target example
 ```
 
+If your workspace contains any Helm unit tests with snapshots (under an `__snapshot__` folder), `tailor` will identify those files are being part of your tests and will also create `helm_unittest_tests` and `resources` targets. See "Snapshot testing" below for more info.
```

Comment:
```suggestion
If your workspace contains any Helm unit tests with snapshots (under a `__snapshot__` folder), `tailor` will identify those files as test snapshots and will create `helm_unittest_tests` and `resources` targets. See "Snapshot testing" below for more info.
```

Reply:
If I understand correctly, `tailor` will create `helm_unittest_tests` even without a `__snapshot__` folder? This reads as if the `helm_unittest_tests` target creation depends on their being a `__snapshots__` folder. But I think those are two separate bits of logic in tailor? Did this mean to say that Pants will infer a dependency from the `helm_unittests_tests` target on the associated `resources` target?

Reply:
Yeah, that's what I meant, unit tests and snapshots are identified by tailor independently and a dependency will be inferred from the former to the later in case snapshots are found.

Reply:
So this needs some rephrasing...